src/clientiface.cpp

```diff
@@ -90,13 +90,27 @@ void RemoteClient::GetNextBlocks (
 		return;
 	}
 
+	// get view range and camera fov from the client
+	s16 wanted_range = sao->getWantedRange();
+	float camera_fov = sao->getFov();
+	// if FOV, wanted_range are not available (old client), fall back to old default
+	if (wanted_range <= 0) wanted_range = 1000;
+	if (camera_fov <= 0) camera_fov = (72.0*M_PI/180) * 4./3.;
```

Comment:
Spaces around '*' and '/'

Reply:
State all numbers as floats
0.0f
72.0f
180.0f
4.0f
3.0f

Reply:
`wanted_range` and `camera_fov` belong each into a separate line.

Reply:
Oh yeah, missed that, agreed.